spec/react_on_rails/generators/install_generator_spec.rb

```diff
@@ -111,4 +111,58 @@
         .to include(GeneratorMessages.format_info(expected))
     end
   end
+
+  context "detect node and npm" do
+    before(:all){ @install_generator = InstallGenerator.new }
+
+    def `(which)
+      ""
+    end
+
+    specify "when node is not exist on *nix" do
+      expect(@install_generator.send(:missing_node?)).to eq true
+    end
+
+    specify "when npm is not exist on *nix" do
+      expect(@install_generator.send(:missing_npm?)).to eq true
+    end
+
+    def `(which)
+      "Not empty string"
+    end
+
+    specify "when node is exist on *nix" do
+       expect(@install_generator.send(:missing_node?)).to eq false
+    end
+
+    specify "when npm is exist on *nix" do
+      expect(@install_generator.send(:missing_npm?)).to eq false
+    end
+
+    RUBY_PLATFORM = "mingw"
+
+    def `(where)
```

Comment:
It was temporary version, now it don't override constant and use stub_constant.
"def `(where)..." -- is the best way i found to make unix work like a windows for test-case. Can't find Rspec method to do system calls more clear.